packages/app-webdir-ui/src/grid/grid.js

```diff
@@ -0,0 +1,87 @@
+// @ts-check
+import PropTypes from "prop-types";
+import React from "react";
+
+/**
+ * @param {object} props
+ * @param {React.ReactNode[]} props.children - Grid items
+ * @param {object[]} props.items - Array of items to render
+ * @param {Function} props.renderItem - Render function for items
+ * @param {number|object} props.columns - Number of columns or responsive columns object
+ * @param {number} props.gap - Bootstrap spacing (0-5)
```

Comment:
`@param {0|1|2|3|4|5} props.gap - Bootstrap spacing (0-5)` typescript will warn if someone tries to pass in a 6